templates/courant/mail.html.twig

```diff
@@ -1,147 +1,392 @@
+
+      .liLink {
+        list-style: none;
+        text-align: center;
+        color: white;
+        padding: 5px;
+      }
+
+      .box {
+        padding: 25px;
+      }
+
+      .table {
+         margin-bottom: 10px;
+         border-radius: 5px;
+      }
+
+      #id {
+        padding: 20px;
+      }
+
+    </style>
+  </head>
+  <body class="">
+      <table>
+      <tr>
+        <td>&nbsp;</td>
+        <td class="container">
+          <div class="content">
+
+            <!-- START CENTERED WHITE CONTAINER -->
+            <table class="main">
+              <tr>
+                <td class="wrapper">
+
+                  <!-- LOGO en TITEL -->
+                  <table border="0" cellpadding="0" cellspacing="0" style="background: #e8e8e8">
+                    <tr style="height: 100px">
+                      <td id="logo" class="inhoud-lb" style="padding: 20px 20px 30px 40px">
+                        <img class="" src="https://upload.wikimedia.org/wikipedia/commons/thumb/7/78/C.S.R._Delft_Logo.svg/190px-C.S.R._Delft_Logo.svg.png" style="width:100px;" />
+                      </td>
+                      <td style="text-align: center; vertical-align: middle; padding-right:20px;">
+                        <hr>
+                          <h1>De courant</h1>
+                        <hr>
+                      </td>
+                    </tr>
+                  </table>
+
+                  <!-- INHOUD -->
+                  <table border="0" cellpadding="0" cellspacing="0" class="table" style="background: #11273a;">
+                  <tr>
+                  {% for categorie,catName in catNames %}
+                    {% if categorie != 'voorwoord' and categorie != 'sponsor' %}
+                      <td class="inhoudKolom">
+                        <div class="inhoudKop">
+                          <h4><u>{{ catName }}</u></h4>
+                        </div>
+                        {% for bericht in berichten %}
+                          {% if bericht.cat.value == categorie %}
+                            <li class="liLink">
+                              <a href="#{{ bericht.id }}" class="link">
+                                {{ bericht.titel | bbcode("mail") }}
+                              </a>
+                            </li>
+                          {% endif %}
+                        {% endfor %}
+                      </td>
+                    {% endif %}
+                  {% endfor %}
+                  </tr>
+                  </table>
+                  
+                  <!--
+                  //Berichten generator//
+                  donker: #11273a
+                  medium: #043763
+                  licht:  #4e698b
+                  -->
+
+                  {% for bericht in berichten %}
+                    {% if loop.index % 4 == 1 %}
+                      {% set bgcolor = '#043763' %}
+                    {% elseif loop.index % 4 == 2 %}
+                      {% set bgcolor = '#4e698b' %}
```

Comment:
Deze bgcolor is een beetje aan de lichte kant: https://contrastchecker.com/?c=ffffff&b=4e698b